cockpit/src/subscriptions-client.js

```diff
@@ -508,19 +508,6 @@ function statusUpdateFailed(reason) {
  * possible status values: https://github.com/candlepin/subscription-manager/blob/30c3b52320c3e73ebd7435b4fc8b0b6319985d19/src/rhsm_icon/rhsm_icon.c#L98
  * [ RHSM_VALID, RHSM_EXPIRED, RHSM_WARNING, RHN_CLASSIC, RHSM_PARTIALLY_VALID, RHSM_REGISTRATION_REQUIRED ]
  */
```

Comment:
This comment could be also deleted, because the old D-Bus API si not used anymore.